pymc3/variational/advi.py

```diff
@@ -182,7 +182,7 @@ def _calc_elbo(vars, model, n_mcsamples, random_seed):
 
     uw = tt.vector('uw')
```

Comment:
Can't the type be forced here already? Or won't the test_value be casted correctly?